src/vs/workbench/contrib/preferences/browser/settingsLayout.ts

```diff
@@ -155,6 +155,11 @@ export const tocData: ITOCEntry = {
 					label: localize('terminal', "Terminal"),
 					settings: ['terminal.*']
 				},
+				{
+					id: 'features/quickopen',
```

Comment:
Looks like I never added Tasks here! Since the setting will be a `task` setting adding quick open here isn't needed. Feel free to either add `task.` here or simple revert the change to this file.